frontend/encore/babel.rst

```diff
@@ -41,7 +41,7 @@ The ``@babel/preset-env`` preset rewrites your JavaScript so that the final synt
 will work in whatever browsers you want. To configure the browsers that you need
 to support, see :ref:`browserslist_package_config`.
 
-After change our "browerslist" config, you will need to manually remove the babel
+After change our "browserslist" config, you will need to manually remove the babel
```

Comment:
There is another issue at the beginning of that sentence ("*After change our*").

Maybe "*After changing your*" or "*After each change to your*" instead?